src/main/java/com/twilio/oai/resolver/common/CodegenModelResolver.java

```diff
@@ -22,6 +22,14 @@ public CodegenModelResolver(IConventionMapper mapper, Map<String, String> modelF
                 languageDataTypes);
```

Comment:
Should update this constructor to call the new one below. Less code.

Reply:
addressed